src/internal/config.ts

```diff
@@ -16,6 +18,20 @@ export const config = {
    */
   onUnhandledError: null as ((err: any) => void) | null,
 
+  /**
+   * A registration point for notifications that cannot be sent to subscribers because they
+   * have completed, errored or have been explicitly unsubscribed. By default, next, complete
+   * and error notifications sent to stopped subscribers are noops. However, sometimes callers
+   * might want a different behavior. For example, with sources that attempt to report errors
+   * to stopped subscribers, a caller can configure RxJS to throw an unhandled error instead.
+   * This will _always_ be called asynchronously on another job in the runtime. This is because
+   * we do not want errors thrown in this user-configured handler to interfere with the
+   * behavior of the library.
+   */
+  onStoppedNotification: undefined as
```

Comment:
nit: I'd default this to `null`. It's the most common thing that such APIs do, and it's what we're doing with `onUnhandledError`.